packages/runtime/test/runtime.test.ts

```diff
@@ -726,4 +727,89 @@ describe('runtime', () => {
     )
     expect(relationsRes).toMatchSnapshot()
   }, 30000)
+
+  test('SET account relation support', async () => {
+    const [favoriteComposite, postComposite] = await Promise.all([
+      Composite.create({ ceramic, schema: favoriteSchema }),
+      Composite.create({ ceramic, schema: postSchema }),
+    ])
+    const favoriteModelID = favoriteComposite.getModelID('Favorite')!
+    const postModelID = postComposite.getModelID('Post')!
+    const composite = Composite.from([favoriteComposite, postComposite], {
+      views: {
+        models: {
+          [postModelID]: {
+            favorites: { type: 'relationFrom', model: favoriteModelID, property: 'docID' },
+          },
+        },
+      },
+    })
+
+    const definition = composite.toRuntime()
+    expect(printGraphQLSchema(definition)).toMatchSnapshot()
+    const runtime = new ComposeRuntime({ ceramic, definition: composite.toRuntime() })
+
+    const createdPosts = await runtime.executeQuery<Record<string, { document: { id: string } }>>(
+      `
+      mutation CreatePosts(
+        $post1Input: CreatePostInput!,
```

Comment:
Is this different from the setProfile in tests case `set profile with custom sync timeout`? Because I'm under the impression we changed the naming convention from `create` to `set` because we now do an upsert, but now I see 'create' profile being used.

Or is it that the `set profile with custom sync timeout` test case has something to do with set relationship type? If thats the case can you explain how.

Reply:
Until now we've been creating `create[Model name]` and `update[Model name]` mutations for all models, whatever account relation type they use.
Now with the SET account relation, instead of `create[Model name]` we have `set[Model name]` which should be more explicit about the behavior and consistent with `unset[Model name]` to reverse/unindex.

For consistency, I think `set[Model name]` should be used as well for the SINGLE account relation because technically that's how it works, it can be treated more as an upsert than an insert, so I added `set[Model name]` for SINGLE account relation models, but kept the `create[Model name]` to avoid making it an immediate breaking change while developers transition to using `set[Model name]` instead. In a future release, we should stop generating these `create[Model name]` mutation for models using the SINGLE account relation.

These changes are only about the SET and SINGLE account relations, for the LIST account relation we still have (and we will continue to have) `create[Model name]` because each document is created uniquely.

Here in the tests, `set profile with custom sync timeout` is testing a model using the SINGLE account relation, while the added test covers the SET account relation. The `Post` model in this test uses the LIST account relation, but the test is mainly about the `Favorite` model which uses the SET account relation. Does that make sense?